src/lib/HsToCoq/ConvertHaskell/Parameters/Edits.hs

```diff
@@ -266,7 +266,8 @@ subtractEdits edits1 edits2 =
   , _replacedTypes                  = edits1^.replacedTypes
   , _collapsedLets                  = edits1^.collapsedLets
   , _inEdits                        = edits1^.inEdits
-  , _exceptInEdits                  = edits1^.exceptInEdits 
+  , _exceptInEdits                  = edits1^.exceptInEdits
+  , _promotions                     = edits1^.promotions
```

Comment:
@ericgiovannini I am merging this PR with `master`. It turns out that I need to add this to make `Promote` edit work with `ExceptIn` edit. Do you think what I did here (in `subtractEdits`) make sense?

Reply:
Yes, this makes sense. Promoting a definition applies to the entire module, so it shouldn't be possible to localize the promote edit to a particular definition or exclude it from applying to a definition. In other words, the `_promotions` field is not used when translating individual definitions. So for `subtractEdits`, which is used when determining the edits that will be applied when translating an individual definition, the value of `_promotions` in the resulting edits record shouldn't matter.

Reply:
Great! Thanks. Merging in now.